lib/lists/helpers.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/**
+ * Get the format of the items in a list
+ * @param {array} item
+ * @returns {string}
+ */
+
+function getFormat(items) {
+  const tester = _.head(items);
+
+  if (_.isString(tester)) {
+    return 'STRING';
+  } else if (_.has(tester,'text')) {
+    return 'OBJECT';
+  } else {
+    log.error('List cannot be modified because it is not a simple list.', { action: 'modifyList' });
```

Comment:
Is this actually an error? Per the spec, lists can be arrays of _arbitrary data_. They must be strings or objects (with `text`) to be used by autocomplete (which also should be documented), but this file is a helper for all lists, not just lists that autocomplete cares about.